app/helpers/fae/application_helper.rb

```diff
@@ -98,7 +98,7 @@ def change_item_link(change)
       if change.changeable_type.exclude?('Fae') && change.changeable_type.exclude?('Page') && (!ActiveRecord::Base.connection.table_exists? change.changeable_type.tableize)
         return "#{change.changeable_type}: model destroyed"
       else
-        text += change.try(:changeable).try(:fae_display_field) || "##{change.changeable_id}"
+        text += change.try(:changeable).try(:fae_display_field).to_s || "##{change.changeable_id}"
```

Comment:
@nilesvm this will break existing functionality. Previously if `change.try(:changeable).try(:fae_display_field)` was `nil` it could use the `change.changeable_id`. But `nil.to_s == ''` so it would never fallback to the ID.

```
>   nil || "something"
 => "something"
> nil.to_s || "something"
 => ""
```

Unfortunately, you'll need to update this to use a conditional.